backend/legacyapi/activity.go

```diff
@@ -89,10 +89,21 @@ type ActivityIssueCreatePayload struct {
 	IssueName string `json:"issueName"`
 }
 
+// TaskRollbackBy records an issue rollback activity.
+// The taskIDs in IssueID are rollbacked by the RollbackByTaskIDs in RollbackByIssueID.
+type TaskRollbackBy struct {
+	IssueID           int   `json:"issueId"`
+	TaskID            []int `json:"taskId"`
```

Comment:
This is a bit confusing when N -> N from taskID to RollbackByTaskID. Maybe we can create a list of activities instead from one task to one task.

Reply:
SGTM
we may combine similar activities when displaying later.